services/encoder/main.go

```diff
@@ -2,25 +2,22 @@ package main
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 
 	log "github.com/sirupsen/logrus"
+	"google.golang.org/protobuf/proto"
 
 	"github.com/Sogilis/Voogle/services/encoder/clients"
-	. "github.com/Sogilis/Voogle/services/encoder/config"
+	"github.com/Sogilis/Voogle/services/encoder/config"
+	"github.com/Sogilis/Voogle/services/encoder/models"
 )
 
 var ctx = context.Background()
 
-type Video struct {
-	Title string `json:"title"`
-}
-
 func main() {
 	log.Info("Starting Voogle encoder")
 
-	config, err := NewConfig()
+	config, err := config.NewConfig()
```

Comment:
Here we have a conflict between the package and the variable. We may want to rename config to cfg